puppet-chrome/index.ts

```diff
@@ -1,16 +1,22 @@
-import IPuppetBrowser from "@secret-agent/puppet-interfaces/IPuppetBrowser";
-import ILaunchedProcess from "@secret-agent/puppet-interfaces/ILaunchedProcess";
-import IPuppetLauncher from "@secret-agent/puppet-interfaces/IPuppetLauncher";
-import os from "os";
-import { Browser } from "./lib/Browser";
-import { Connection } from "./lib/Connection";
+import IPuppetBrowser from '@secret-agent/puppet-interfaces/IPuppetBrowser';
+import ILaunchedProcess from '@secret-agent/puppet-interfaces/ILaunchedProcess';
+import IPuppetLauncher from '@secret-agent/puppet-interfaces/IPuppetLauncher';
+import os from 'os';
+import Path from 'path';
```

Comment:
Shouldn't this be `import * as Path from 'path'`?

Reply:
Only in strict mode, which we don't have turned on in SA.

Reply:
What standard should we follow in our code? It seems we have it both ways in the code base.

Reply:
Good question. I didn't realize how split it was.